addons/im_livechat/demo/im_livechat_channel/im_livechat_support_bot_session_6.xml

```diff
@@ -54,5 +54,66 @@
+        </record>
+        <record id="support_bot_session_6_chatbot_message_3" model="chatbot.message">
+            <field name="mail_message_id" ref="support_bot_session_6_message_4"/>
+            <field name="discuss_channel_id" ref="support_bot_session_6_demo"/>
+            <field name="script_step_id" ref="chatbot_script_initial_step_demo"/>
+            <field name="user_script_answer_id" ref="chatbot_script_answer_discuss_demo"/>
+            <field name="user_raw_script_answer_id" ref="chatbot_script_answer_discuss_demo"/>
+        </record>
+        <record id="support_bot_session_6_message_5" model="mail.message">
+            <field name="model">discuss.channel</field>
+            <field name="res_id" ref="support_bot_session_6_demo"/>
+            <field name="message_type">comment</field>
+            <field name="author_id" ref="support_bot_operator_partner_demo"/>
+            <field name="body">Let me check if I can connect you with the right expert...</field>
+            <field name="subtype_id" ref="mail.mt_comment"/>
+            <field name="date" eval="DateTime.today() + relativedelta(days=-6, minutes=5)"/>
+        </record>
+        <record id="support_bot_session_6_member_agent_demo" model="discuss.channel.member">
+            <field name="partner_id" ref="base.partner_admin"/>
+            <field name="channel_id" ref="support_bot_session_6_demo"/>
+            <field name="livechat_member_type">agent</field>
+            <field name="agent_expertise_ids" eval="[(4, ref('im_livechat_chatbot_script_expertise_discuss_demo'))]"/>
+        </record>
+        <record id="support_bot_session_6_message_6" model="mail.message">
+            <field name="model">discuss.channel</field>
+            <field name="res_id" ref="im_livechat.support_bot_session_6_demo"/>
+            <field name="author_guest_id" ref="im_livechat.support_bot_session_6_guest_demo"/>
+            <field name="author_id"/>
+            <field name="subtype_id" ref="mail.mt_note"/>
+            <field name="message_type">notification</field>
+            <field name="date" eval="DateTime.today() + relativedelta(days=-6, minutes=6)"/>
+        </record>
+        <record id="support_bot_session_6_rating_message_demo" model="rating.rating">
+            <field name="access_token">SUPPORT_BOT_RATTING_1</field>
+            <field name="res_id" ref="im_livechat.support_bot_session_6_demo"/>
+            <field name="res_model_id" ref="mail.model_discuss_channel"/>
+            <field name="message_id" ref="support_bot_session_6_message_6"/>
+            <field name="rated_partner_id" ref="base.partner_demo"/>
+            <field eval="False" name="partner_id"/>
+            <field eval="True" name="consumed"/>
+        </record>
+        <function model="discuss.channel" name="rating_apply"
+            eval="([ref('im_livechat.support_bot_session_6_rating_message_demo')], 5, 'SUPPORT_BOT_RATTING_1', None, 'Good Job!')"/>
```

Comment:
Ratingis irrelevant in this flow, and it actually extends the duration of the session (since rating will be created based on current date while the conversation is 6 days old), leading to incoherent demo data.